tensorflow/python/ops/image_grad.py

```diff
@@ -153,3 +153,221 @@ def _CropAndResizeGrad(op, grad):
       grad, op.inputs[0], op.inputs[1], op.inputs[2])
 
   return [grad0, grad1, None, None]
+
+
+def _CustomReciprocal(x):
```

Comment:
After the most recent batch of changes, the docstring for this function no longer matches the arguments. It's probably best to drop this function definition and replace the calls to `_CustomReciprocal()` below with direct calls to `math_ops.div_no_nan`

Reply:
Resolved in 032bdc6.